administrator/components/com_associations/Helper/AssociationsHelper.php

```diff
@@ -286,33 +322,150 @@ public static function getAssociationHtmlList($extensionName, $typeName, $itemId
 					$additional = '<strong>' . Text::sprintf('COM_MENUS_MENU_SPRINTF', $menutype_title) . '</strong><br>';
 				}
 
-				$labelClass  = 'badge-secondary';
+				$labelClass  = 'badge-success';
 				$target      = $langCode . ':' . $items[$langCode]['id'] . ':edit';
 				$allow       = $canEditReference
 								&& self::allowEdit($extensionName, $typeName, $items[$langCode]['id'])
 								&& self::canCheckinItem($extensionName, $typeName, $items[$langCode]['id']);
 
-				$additional .= $addLink && $allow ? Text::_('COM_ASSOCIATIONS_EDIT_ASSOCIATION') : '';
+				$parentChildInfoSpace = $additional && !$addLink ? '<br>' : '<br><br>';
+
+				if ($defaultAssocLang)
+				{
+					// Settings for the default association language.
+					if ($defaultAssocLang === $langCode)
+					{
+						$labelClass .= ' parent-item';
+						$additional .= $addLink && $allow ? Text::_('COM_ASSOCIATIONS_EDIT_ASSOCIATION') : '';
+						$parentChildInfo  = $parentChildInfoSpace . Text::_('JGLOBAL_ASSOCIATIONS_DEFAULT_ASSOC_LANG_ITEM');
+
+						if ($defaultAssocLang === $itemLanguage)
+						{
+							// Do not define any child target as there can be more than one
+							$target = '';
+						}
+						else
+						{
+							$target = $itemLanguage . ':' . $itemId . ':edit';
+						}
+					}
+					// Setting for children
+					else
+					{
+						// When there is no associated parent, set it to target
+						if (!$parentId)
+						{
+							$target        = $defaultAssocLang . ':0:add';
+						}
+
+						if (array_key_exists($items[$langCode]['id'], $assocParentDates) && array_key_exists($parentId, $assocParentDates))
+						{
+							$associatedModifiedParent = $assocParentDates[$items[$langCode]['id']][0];
+							$lastModifiedParent       = $assocParentDates[$parentId][0];
+
+							if ($associatedModifiedParent < $lastModifiedParent)
+							{
+								// Don't display not corresponding item
+								if ($assocState !== 'all' && $assocState !== 'out_of_date')
+								{
+									unset($items[$langCode]);
+									continue;
+								}
+
+								$additional .= $addLink && $allow ? Text::_('COM_ASSOCIATIONS_UPDATE_ASSOCIATION') : '';
+								$labelClass    = 'badge-warning';
+								$target = $langCode . ':' . $items[$langCode]['id'] . ':edit';
+								$update = true;
+
+								/*
+								When versions are disabled then the modified date is used for the parent.
+								That means that when no changes were made and the parent has been saved the modified date has been changed.
+								So the out-of-date state means in that case there might have been made changes and it is necessary to check manually and update the target.
+								*/
```

Comment:
https://developer.joomla.org/coding-standards/inline-code-comments.html
```suggestion
								/**
								 * When versions are disabled then the modified date is used for the parent.
								 * That means that when no changes were made and the parent has been saved the modified date has been changed.
								 * So the out-of-date state means in that case there might have been made changes and it is necessary to check manually and update the target.
								 */
```